src/conversions/array.rs

```diff
@@ -213,6 +231,19 @@ mod tests {
             assert!(&v == b"abc");
         });
     }
+    #[test]
+    fn test_topyobject() {
```

Comment:
nit: This should have a more descriptive name.

Reply:
Should I change this and force push, or just add a new commit?

Reply:
I don't mind either way, but I'd lean towards force pushing.

Reply:
Naming things is hard, I hope you like the new one better :^)